src/frequenz/sdk/timeseries/battery_pool/_component_metric_fetcher.py

```diff
@@ -120,6 +126,12 @@ async def fetch_next(self) -> ComponentMetricsData | None:
             data = await asyncio.wait_for(
                 self._receiver.receive(), self._max_waiting_time
             )
+            if _missing_data_logger.is_limiting():
+                _missing_data_logger.reset()
+                _missing_data_logger.debug(
+                    "Component %d has started sending data.", self._component_id
+                )
+                _missing_data_logger.reset()
```

Comment:
Couldn't you just log using the regular `_logger` only `reset()` here?
```suggestion
                __logger.debug(
                    "Component %d has started sending data.", self._component_id
                )
                _missing_data_logger.reset()
```